### PR TITLE
Issues about CommentIndicators that happened during serialization

### PR DESCRIPTION
**Pull request by Paula-Kli**
closes https://github.com/hpi-swa-lab/BP2020RH1/issues/96

### CODE REVIEW

File: lively.collab/comments/commentBrowser.js [hunk 1]
```diff
@@ -184,8 +186,21 @@ export class CommentBrowser extends Window {
...
     this.container.addMorph(newContainer);
-    containerToRemove.submorphs.forEach((commentGroup) => commentGroup.hideCommentIndicators());
-    newContainer.submorphs.forEach((commentGroup) => commentGroup.showCommentIndicators());
+    this.hideAllCommentIndicators();
+    this.showCommentIndicators();
+  }
+
+  showCommentIndicators () {
+    if (this.showsResolvedComments) {
+      this.resolvedCommentContainer.submorphs.forEach((commentGroup) => commentGroup.showCommentIndicators());
```
Comment (linusha) [inline lively.collab/comments/commentBrowser.js:195]: Please use `withAllSubmorphsDo` to iterate over submoprhs.

**Review by linusha — CHANGES_REQUESTED** [review]

File: lively.collab/comments/commentBrowser.js (hunk 1 above)
Comment (frcroth) [reply lively.collab/comments/commentBrowser.js:195]: Please don't

Comment (Paula-Kli) [reply lively.collab/comments/commentBrowser.js:195]: we don't want to do it with all submorphs but only with the first hierarchy line

Comment (linusha) [reply lively.collab/comments/commentBrowser.js:195]: .

**Review by linusha — APPROVED** [review]

**Review by frcroth — APPROVED** [review]

**Pull request merged**